test/test.js

```diff
@@ -43,8 +43,6 @@ test('getPrecision', function(t) {
 test('equality', function(t) {
 	t.ok(number('13').equal('13.000'))
 	t.notOk(number('13').equal('13.0000000001'))
-	t.ok(number('13.000').equal('13'))
-	t.notOk(number('13.0000000001').equal('13'))
```

Comment:
I'd prefer to leave these - it's true that `number('13.000').equal('13')` *should* be equivalent to `number('13').equal('13.000')`, but I like asserting it anyway for sanity's sake.